operations/deployment-engineering/playbook-mme-sev1-outages.md

```diff
@@ -0,0 +1,182 @@
+# Deployment Engineering: Playbook for MME Sev 1 Outages
+
+Below is a codified playbook used to respond to MME Sev 1 Outages.
```

Comment:
maybe there should be a link to the playbook in community, as it would be the source of truth

Reply:
Thanks! Great feedback, added a link to the playbook in community as source of truth